docs/background_jobs.md

```diff
@@ -0,0 +1,21 @@
+# Background jobs
+
+Sidekiq is the background job processor we use that handles long-running tasks, such as sending emails or processing images, asynchronously.
+
+`AttachProfilePictureWorker` is a Sidekiq worker that downloads an image from a given URL and attaches it to the corresponding user record.
+
```

Comment:
mention about how on deployment they need to have another worker for the sidekiq for that process to run and sidekiq to process the queue. also need to mention how redis is required.